repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -169,74 +169,54 @@ export const getGeoZoneFareTable = (
                     },
                 },
-            },
-        },
-        includes: {
-            FareTable: {
-                version: '1.0',
-                id: `op:${product.productName}@${userPeriodTicket.zoneName}@p-ticket`,
-                Name: { $t: `${product.productName}` },
-                pricesFor: {
-                    SalesOfferPackageRef: {
-                        version: '1.0',
-                        ref: `Trip@${ticketUserConcat}-${product.productName}-SOP@${product.salesOfferPackages[0].name}`,
-                    },
-                    ...getCarnetQualityStructureFactorRef(product),
-                },
-                specifics: {
-                    TypeOfTravelDocumentRef: {
-                        version: '1.0',
-                        ref: 'op:p-ticket',
-                    },
-                },
                 columns: {
                     FareTableColumn: {
                         version: '1.0',
-                        id: `op:${product.productName}@${userPeriodTicket.zoneName}@p-ticket`,
-                        Name: { $t: `${product.productName}` },
+                        id: `op:${product.productName}@${userPeriodTicket.zoneName}`,
+                        Name: { $t: `${userPeriodTicket.zoneName}` },
                         representing: {
-                            TypeOfTravelDocumentRef: {
+                            TariffZoneRef: {
                                 version: '1.0',
-                                ref: 'op:p-ticket',
+                                ref: `op:${placeHolderGroupOfProductsName}@${userPeriodTicket.zoneName}`,
                             },
-                            ...profileRef,
                         },
                     },
                 },
                 includes: {
                     FareTable: {
                         version: '1.0',
-                        id: `op:${product.productName}@${userPeriodTicket.zoneName}@p-ticket@${userPeriodTicket.passengerType}`,
-                        Name: { $t: `${product.productName} - ${userPeriodTicket.passengerType}` },
-                        limitations: {
-                            ...profileRef,
+                        id: `op:${product.productName}@${userPeriodTicket.zoneName}@p-ticket`,
+                        Name: { $t: `${product.productName}` },
+                        pricesFor: {
+                            SalesOfferPackageRef: {
+                                version: '1.0',
+                                ref: `Trip@${ticketUserConcat}-${product.productName}-SOP@${salesOfferPackage.name}`,
+                            },
+                            ...getCarnetQualityStructureFactorRef(product),
+                        },
+                        specifics: {
+                            TypeOfTravelDocumentRef: {
+                                version: '1.0',
+                                ref: 'op:p-ticket',
```

Comment:
I think that this reference need to reference the ticket media of the ticket, as here it is hard coded to paper ticket. If there are multiple, I expect it will need to make a TypeOfTravelDocumentRef for each media type.